python/tskit/util.py

```diff
@@ -534,17 +534,29 @@ def html_table(rows, *, header):
     """
 
 
+def metadata_codec(table):
+    if hasattr(table, "metadata_schema"):
+        schema = table.metadata_schema.schema
+        return "raw" if schema is None else schema.get("codec", "unknown")
```

Comment:
Looks like the tests don't check if this value is correct? (eg, no examples of "unknown" codec)